src/applications/ivc-champva/10-7959C/chapters/healthInsuranceInformation.js

```diff
@@ -46,211 +47,295 @@ schemas
 */
 export function applicantHasInsuranceSchema(isPrimary) {
   const keyname = isPrimary ? 'applicantHasPrimary' : 'applicantHasSecondary';
-  const property = isPrimary ? 'hasPrimary' : 'hasSecondary';
   return {
     uiSchema: {
-      applicants: { items: {} },
-    },
-    schema: applicantListSchema([], {
+      ...titleUI(
+        ({ formData }) =>
+          `${nameWording(formData)} ${
+            isPrimary ? 'primary' : 'secondary'
+          } health insurance`,
+      ),
       [keyname]: {
-        type: 'object',
-        properties: {
-          [property]: { type: 'string' },
-          _unused: { type: 'string' },
-        },
+        ...yesNoUI({
+          updateUiSchema: formData => {
+            const useFirstPerson = formData.certifierRole === 'applicant';
+            return {
+              'ui:title': `${
+                useFirstPerson
+                  ? 'Do you'
+                  : `Does ${nameWording(formData, false)}`
+              } need to provide or update any other health insurance coverage?`,
+              'ui:options': {
+                hint: additionalFilesHint,
+              },
+            };
+          },
+        }),
       },
-    }),
+    },
+    schema: {
+      type: 'object',
+      required: [keyname],
+      properties: {
+        [keyname]: yesNoSchema,
+      },
+    },
```

Comment:
This type of change is carried all through this file. Key points are:
- Remove list-loop specific config `applicants: { items {...} }`
- Remove CustomPage specific config `[property]: { type: 'string'} ...`
- Add in the standard UI/Schema items to replace custom page functionality (e.g., `yesNoUI`)
- Handle creating dynamic header text (formerly the job of the CustomPage) using `updateUiSchema`